cohort_4/playbook/activities.md

```diff
@@ -24,6 +29,51 @@
 
 TODO
 
+## Brainstorming
+
+### Forming groups
+
+- Assign people to groups.
+  - There are social pressures between friends when forming groups. Start by
+    assigning the hackers to random groups.
+  - If any problems arise in a group handle it on a case-by-case basis.
+
+### Yes-and sessions
+
+- In a "Yes-and" session, pairs of hackers build off each-others' ideas to create
+  long, completely nonsensical ideas
+  - "Yes-and" sessions are exercises that help people get out of a judgmental
+    mindset before brainstorming
+- To run a "Yes-and" session, start with an example between facilitators
+  - ex: between person A and person B:
+
+    ```
+    A: Let's create tinder for unicorns
+    B: And it will be super effective, and the unicorn population will increase 1,000 times
+    A: Yes, and there won't be enough food for the unicorns
+    B: So they'll build a spaceship to search for new planets to colonize!
+    ```
+
+- Have the hackers jump into a "Yes-and session" for 2 minutes then let
+  volunteer to present the ideas they came up with.
```

Comment:
replace "present the ideas they came up with." with "present the stories that unfolded". Could be confusing to think that the ideas for project are to be thought of during the yes and sesion